Makefile

```diff
@@ -92,21 +99,32 @@ azuredisk-windows:
 	if [ ! -d ./vendor ]; then dep ensure -vendor-only; fi
 	CGO_ENABLED=0 GOOS=windows go build -a -ldflags ${LDFLAGS} -o _output/azurediskplugin.exe ./pkg/azurediskplugin
 
-.PHONY: container
-container: azuredisk
-	docker build --no-cache -t $(IMAGE_TAG) -f ./pkg/azurediskplugin/Dockerfile .
-
 .PHONY: azuredisk-container
-azuredisk-container: azuredisk
+azuredisk-container:
+ifdef CI
+	az acr login --name $(REGISTRY_NAME)
+	make azuredisk azuredisk-windows
+	az acr build --registry $(REGISTRY_NAME) -t $(IMAGE_TAG)-linux-amd64 -f ./pkg/azurediskplugin/Dockerfile --platform linux .
+	az acr build --registry $(REGISTRY_NAME) -t $(IMAGE_TAG)-windows-1809-amd64 -f ./pkg/azurediskplugin/Windows.Dockerfile --platform windows .
+	docker manifest create $(IMAGE_TAG) $(IMAGE_TAG)-linux-amd64 $(IMAGE_TAG)-windows-1809-amd64
+	docker manifest inspect $(IMAGE_TAG)
+else
+ifdef TEST_WINDOWS
+	make azuredisk-windows
+	docker build --no-cache -t $(IMAGE_TAG) -f ./pkg/azurediskplugin/Windows.Dockerfile .
+else
+	make azurediskfi
```

Comment:
what is `azurediskfi`? typo?

Reply:
It's a typo. Let me open another PR to fix that.